### ISSUE TITLE
Draft page: player selection queue

### ISSUE THREAD

**Issue by TheAdameia** [TheAdameia/Fantasy-Football-Golf-Scoring#35]
Given a user is viewing a live draft,
When a user selects a player to add to their auto-draft selection queue,
The player should be added to their selection queue.

Prerequisites: #33, #32

Moved to low priority to simplify the task of implementing SignalR for the live draft page.
